addon/ng2/utilities/completion.sh

```diff
@@ -1,18 +1,25 @@
-###-begin-ng-completion### 
+###-begin-ng-completion###
 #
 # ng command completion script
 #
 # Installation: ng completion >> ~/.bashrc (or ~/.zshrc)
 #
 
-ng_opts='new init build serve generate autocomplete e2e lint test version'
-init_opts='--dry-run --verbose --blueprint --skip-npm --skip-bower --name'
-new_opts='--dry-run --verbose --blueprint --skip-npm --skip-bower --skip-git --directory'
-build_opts='--environment --output-path --watch --watcher'
-serve_opts='--port --host --proxy --insecure-proxy --watcher --live-reload --live-reload-host
-            --live-reload-port --environment --output-path --ssl --ssl-key --ssl-cert'
-generate_opts='component directive pipe route service'
-test_opts='--watch --browsers --colors --log-level --port --reporters'
+ng_opts='addon asset-sizes b build completion d destroy doc e2e g generate get github-pages:deploy gh-pages:deploy h help i init install lint make-this-awesome new s serve server set t test v version -h --help'
```

Comment:
`asset-sizes d destroy`, we do not support these commands currently. They might work, somewhat, but we do not support them.